AEPIdentity/Tests/IdentityStateTests.swift

```diff
@@ -632,6 +633,42 @@ class IdentityStateTests: XCTestCase {
         XCTAssertEqual(hitResponse.ttl, state.identityProperties.ttl) // ttl should have been updated
     }
 
+    /// Tests that when a non-opt out response is handled with a non-matching ECID that we don't update the last sync and other identity properties, along with dispatching two identity events
```

Comment:
let's add a comment here like:
` This situation can usually happen if a network response is handled at the same time as the resetIdentities request.`